c-cpp/mixbox.c

```diff
@@ -693,13 +707,7 @@ static int decompress(char* obuffer, int olen)
 
 INLINE static const unsigned char* mixbox_lut()
 {
-  struct mixbox_init_t
-  {
-    unsigned char lut[64*64*64*3 + 12675];
-    mixbox_init_t() { decompress((char*)lut, sizeof(lut)); }
-  };
-
-  static const mixbox_init_t decompressed;
-
-  return decompressed.lut;
+  static unsigned char lut[64*64*64*3 + 12675];
+  decompress((char*)&lut[0], sizeof(lut));
```

Comment:
This will call `decompress()` on every call to `mixbox_lut()`, whereas the original code only calls `decompress()` once.